src/frontend/src/pages/Auth/Login.tsx

```diff
@@ -48,6 +49,17 @@ export default function Login() {
     }
 
     checkLoginState(navigate, location?.state?.redirectFrom, true);
+
+    // check if we got login params (login and password)
+    if (searchParams.has('login') && searchParams.has('password')) {
+      console.log('got login params');
```

Comment:
Please remove this debug msg :)

Reply:
I thought this might be helpful when debugging the demo instance but I have removed it now